.changeset/pretty-dingos-travel.md

```diff
@@ -0,0 +1,14 @@
+---
+"@frontity/components": minor
+"@frontity/comscore-analytics": minor
+"@frontity/connect": minor
+"@frontity/core": minor
+"frontity": minor
+"@frontity/google-ad-manager": minor
+"@frontity/google-analytics": minor
+"@frontity/google-tag-manager-analytics": minor
+"@frontity/html2react": minor
+"@frontity/smart-adserver": minor
```

Comment:
I understand that `@frontity/core` needs a minor bump, but why the rest?

Reply:
The bunch of them have updates to `react-test-renderer` and in changeset I selected all of them. That's all there is. Should I switch to patch instead? Might be better since the updates are not affecting the user-facing code, only the internal. Gonna patch them.

Reply:
> Might be better since the updates are not affecting the user-facing code, only the internal

Actually, we don't release a new version when updates don't affect user-facing code.

There is one exception, which is changes to the readme file because if we don't release a new version the readme of NPM is not updated. So I think you can remove them from the changeset and leave `@frontity/core` alone.

Reply:
I'm going to request the change then 🙂